tests/sles4sap/sapconf.pm

```diff
@@ -20,7 +20,7 @@ use strict;
 use warnings;
 
 sub run_developers_tests {
-    my $devel_repo = 'https://gitlab.suse.de/AngelaBriel/sapconf-test/repository/master/archive.tar.gz';
+    my $devel_repo = 'https://gitlab.suse.de/AngelaBriel/sapconf-test/repository/sapconf_v5/archive.tar.gz';
```

Comment:
Didn't noticed before, but I think that this could be a test variable instead, easier to change next time.

Reply:
> Didn't noticed before, but I think that this could be a test variable instead, easier to change next time.

I don't believe that's necessary because there will be one branch in the end.

Reply:
And when the repo will be moved? Or if Angela change job/company?

Reply:
> And when the repo will be moved? Or if Angela change job/company?

Is this really a problem?  I already forked the repo.

My point is that this won't change that often so I really don't see the need for more variables.